### PR TITLE
imgcat 2.9 (new formula)

### PR DESCRIPTION
**Pull request by anapsix**
### All Submissions:

- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) for the same update/change?

_You can erase any parts of this template not applicable to your Pull Request._

### New Formulae Submissions:

- [x] Does your submission pass
`brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Have you built your formula locally prior to submission with `brew install <formula>`?

### CODE REVIEW

**Comment by nijikon** [comment]
@BrewTestBot test this please

File: Library/Formula/imgcat.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class Imgcat < Formula
+  desc "iTerm's imgcat"
+  homepage "https://www.iterm2.com/images.html"
+  url "https://raw.githubusercontent.com/gnachman/iTerm2/7fb596f04b8e8c7e730efe1a033317f121a2f943/tests/imgcat"
```
Comment (dunn) [inline Library/Formula/imgcat.rb]: We need a stable, tagged release.

**Comment by DomT4** [comment]
@nijikon The failure here won't be resolved by a CI re-queue. It's a Jenkins thing.

File: Library/Formula/imgcat.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  version "0.1"
+  sha256 "036ee8aec2487a02b40d62d72918090ae022fc018589337dfd8403e08ffdd0c0"
+
+  def install
+    bin.install "imgcat"
+  end
+
+  test do
+    system "/usr/bin/curl", "-sSL", "http://imgs.xkcd.com/comics/security.png | imgcat"
```
Comment (dunn) [inline Library/Formula/imgcat.rb]: We have a test PNG in the `test/fixtures` directory.

Comment (bfontaine) [inline Library/Formula/imgcat.rb]: See [`catimg.rb`](https://github.com/Homebrew/homebrew/blob/e8a2f30bffcae384bacef55fd1fe58f074cb7543/Library/Formula/catimg.rb#L23) for an example.

**Comment by anapsix** [comment]
- updated version to match version of iTerm, since it's just a script from iTerm's repo
- switched to using test image from fixtures

**Comment by DomT4** [comment]
Going to close this. I don't think it's really going to work in a way that's suitable for Homebrew; there's no separate release process, there's no versioning as such, there's no guarantee upstream won't rewrite it in a breaking way or remove it entirely, etc.

It's essentially just a bash script, which means you can stick this in a tap you control and support it there with no change in usage from how you'd use it in Homebrew. You can find documentation on that [here](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/How-to-Create-and-Maintain-a-Tap.md) if you'd like to go down that route.

Thank you for the submission to Homebrew nonetheless, we do appreciate that.

**Comment by anapsix** [comment]
@DomT4 pardon the noise then.
I'm new to Homebrew process and wasn't sure what's the best way to add something like this.. Cheers!

**Pull request closed without merging**